lib/constructs/interface.js

```diff
@@ -55,6 +56,15 @@ Interface.prototype.generateConstructor = function () {
     }
     this.str += conversions.body + "\n";
 
+    if (utils.getExtAttr(this.idl.extAttrs, "JSDOMEventTarget")) {
+      this.str += `
+  Object.defineProperty(this, "_ownerDocument", {
```

Comment:
This will expose `_ownerDocument` on the wrapper, yes? We definitely want to avoid that. Can we just provide it to the impl in the constructor?